src/server/models/listItem/providers/helpers.ts

```diff
@@ -0,0 +1,232 @@
+import {
+  Address,
+  CountryName,
+  ListItem,
+  Point,
+  ServiceType,
+} from "server/models/types";
+import pgescape from "pg-escape";
+import { geoPointIsValid } from "server/models/helpers";
+import { ROWS_PER_PAGE } from "server/models/listItem";
+import { prisma } from "server/models/db/prisma-client";
+import { get, startCase, toLower } from "lodash";
+import { logger } from "server/services/logger";
+import {
+  CovidTestSupplierFormWebhookData,
+  LawyersFormWebhookData,
+} from "server/components/formRunner";
+import { UpdatableAddressFields } from "server/models/listItem/providers/types";
+
+/**
+ * Constructs SQL for querying published list items.  If the region is not populated
+ * or is set to "Not set" then it will be ordered by company name otherwise by distance
+ * from the geo point.
+ * @param props
+ */
+export function fetchPublishedListItemQuery(props: {
```

Comment:
point made in a previous comment about using a generic "fetch" function to avoid duplicating SQL in multiple places.

Reply:
This is old code, just moved, otherwise this PR would've taken forever. Although I agree. Most of these queries can be done with prisma.